src/main/java/org/apache/sysml/hops/ipa/InterProceduralAnalysis.java

```diff
@@ -460,8 +460,22 @@ private void determineFunctionCandidatesNNZPropagation(Map<String, FunctionOp> f
 	/////////////////////////////
 	// INTRA-PROCEDURE ANALYSIS
 	//////	
-	
-	private void propagateStatisticsAcrossBlock( StatementBlock sb, Map<String, Integer> fcand, LocalVariableMap callVars, Map<String, Set<Long>> fcandSafeNNZ, Set<String> unaryFcands, Set<String> fnStack ) 
+
+	/**
+	 * Perform intra-procedural analysis (IPA) by propagating statistics
+	 * across statement blocks.
+	 *
+	 * @param sb  DML statement blocks.
+	 * @param fcand  Function candidates.
+	 * @param callVars  Calling program's map of variables eligible for
+	 *                     propagation.
+	 * @param fcandSafeNNZ  Function candidate safe non-zeros.
+	 * @param unaryFcands  Unary function candidates.
+	 * @param fnStack  Function stack to determine current scope.
+	 * @throws HopsException
+	 * @throws ParseException
```

Comment:
I know any description for the exceptions is absolutely redundant but probably better than a javadoc warning